components/director/cmd/main.go

```diff
@@ -158,3 +165,18 @@ func configureLogger() {
 	})
 	log.SetReportCaller(true)
 }
+
+func getTenantMappingHanderFunc(transact persistence.Transactioner, staticUsersSrc string) func(writer http.ResponseWriter, request *http.Request) {
+	if v := os.Getenv("TENANT_MAPPING_MOCK"); v != "" {
```

Comment:
please define it in `config` or delete - because maybe we don't need it at all

Reply:
Don't get why we should treat toggles as configuration. But I decided to remove the previous implementation so this won't be needed anymore.